src/Symfony/Component/Dotenv/Dotenv.php

```diff
@@ -402,7 +402,6 @@ private function resolveCommands($value)
             }
 
             $process = method_exists(Process::class, 'fromShellCommandline') ? Process::fromShellCommandline('echo '.$matches[0]) : new Process('echo '.$matches[0]);
-            $process->inheritEnvironmentVariables(true);
```

Comment:
same here about the min version.

Reply:
btw, as this is not a requirement, it might be worth making it conditional (based on the detection of `method_exists(Process::class, 'fromShellCommandline')` done for the initialiazation too, for instance)

Reply:
Done thanks.
For WebServer and Dotenv, kept the current versions but used the method you've hint to call this method or not.
For VarDumper, Process is only used for testing purpose. Let's bump.